### ISSUE TITLE
Anti-sp100 antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6595]
**Preferred term label:**Anti-sp100 antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Anti-sp100 antibodies are directed against sp100 antigen found within nuclear bodies;   large protein complexes in the nucleus that may have a role in cell growth and differentiation. Anti-sp100 antibodies are found in approximately 20–30% of patients diagnosed with primary biliary cirrhosis (PBC). PMID 12848948

**Parent term (use hpo.jax.org/app)**
Antinuclear antibody positivity HP:0003493

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Biliary Cirrhosis, Primary OMIM:109720

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik 
Should we make this a synonym or child of this 
https://hpo.jax.org/app/browse/term/HP:0025353

**Comment by eswietlik** [comment]
@pnrobinson Yes, it makes sense.

**Comment by pnrobinson** [comment]
done

**Comment by eswietlik** [comment]
Correct, they show nuclear dots pattern in IIF on Hep cells.
E

sob., 23 sty 2021 o 12:09 Peter Robinson <notifications@github.com>
napisał(a):
